packaging/suse/portus.spec.in

```diff
@@ -49,20 +59,37 @@ Obsoletes:      Portus < %{version}
 # This is the reason why we are obsoleting it
 Obsoletes:      Portus = 20151120162040
 
-# Javascript engine to build assets
+# Javascript engine to build assets. Note that yarn-packaging will automatically
+# create the provides for the JS libs
 BuildRequires:  nodejs6
 BuildRequires:  yarn
+BuildRequires:  yarn-packaging
 
 # Base ruby engine.
 %define rb_build_versions ruby25
 %define rb_build_abi      ruby:2.5.0
 %define rb_suffix         ruby2.5
 %define rb_ver            2.5.0
-BuildRequires:  %{rubydevel}
-BuildRequires:  %{rubygem gem2rpm}
 Requires:       config(%{rb_suffix}) >= %{rb_default_ruby_abi}
 
-__RUBYGEMS_BUILD_REQUIRES__
+BuildRequires: libcurl-devel
+Requires: libcurl4
+BuildRequires: libffi-devel
+%if 0%{?suse_version} <= 1320
+BuildRequires: libmysqlclient-devel < 10.1
+Requires: libmysqlclient18 < 10.1
+%else
+BuildRequires: libmysqlclient-devel
+Requires: libmysqlclient18
+%endif
+Recommends: mariadb
```

Comment:
I'd drop this recommend, to be honest. We are recommending containerized deployments, and in that scenario this doesn't make sense.